### ISSUE TITLE
Split each pixel of carpet plot into two triangles (for primary and secondary data reference)

### ISSUE THREAD

**Issue by hdrake** [hdrake/cmip6hack-multigen#21]
As in:
<img width="457" alt="Screen Shot 2019-10-16 at 9 24 19 AM" src="https://user-images.githubusercontent.com/12971166/66923711-53004d00-eff7-11e9-8224-20a00cb68678.png">

**Comment by bradyrx** [comment]
Is the carpet plot just a necessity of this kind of study? There's just so much info packed into this and I feel like the two triangles makes it harder to read. I guess though having way less variables and models helps us here.

Maybe there's some method to reduce the dimensionality of all these variables (on the y-axis) into one score for each model. I'm not sure of the literature/field to know if there's some standard method there.

Another resource to check is how ILAMB handles their inter-model comparison (https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2018MS001354). Nathan has spent years developing this package, so they probably have the best metric(s) there are for something like this. This leads to an absolute and relative score for each model/variable:

<img width="868" alt="Screen Shot 2019-10-27 at 5 04 06 PM" src="https://user-images.githubusercontent.com/8881170/67643323-dfeeb480-f8db-11e9-9ce8-59765718d5c4.png">

**Comment by hdrake** [comment]
> Maybe there's some method to reduce the dimensionality of all these variables (on the y-axis) into one score for each model. I'm not sure of the literature/field to know if there's some standard method there.

Yes, you can define something like a "performance index", which is usually the equivalent of an average of the relative scores for all variables. I think there's value in both kinds of plots for large numbers of variables. If we're only going to do temperature and precipitation, however, we can probably just skip the carpet plots all together and do a <a href="https://github.com/hdrake/cmip6hack-multigen/blob/master/figures/model_performance_over_time.png">scatter plot panel for each variable</a>.

**Comment by bradyrx** [comment]
Yes that sounds like a good way to avoid the carpet plots. Although the "carpet plots" we do have are fine and look nice since there's only a few models and variables. It'd be easier to handle https://github.com/hdrake/cmip6hack-multigen/issues/16 with the scatter plot panel though.
